### ISSUE TITLE
23281163: UISegmentedControl does not support Accessibility (Labels/Hints) for images in segments

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6540]
#### Description

Summary:
When using UISegmentedControl and setting it up with UIImages as contents for each segment... this thing becomes inaccessible by users which rely on VoiceOver support.

Even attaching accessibilityLabel to the UIImage does not change this. VoiceOver ignores this.

Steps to Reproduce:
Try to find out which segment of a UISegmentedControl – which is setup with images – is actually selected using VoiceOver. You will ony hear the segment index, no further accessibility info.

That is beacuse you cannot provide accessibility info for UISegmentedControls.

Expected Results:
I can apply accessibilityLabel & accessibilityHint to each segment or at least to the UIImges used and they will be respected by VoiceOver.

Actual Results:
That is impossible, therefore segments carrying image content are inaccessible by users rellying on VoiceOver.

Version:
any iOS qualifies, i.e. iOS 9.1 (13B143)

Notes:

Configuration:
any device running iOS will qualify

Attachments:
'UISegmentedControl.PNG' was successfully uploaded.
## 

Product Version: any iOS qualifies, i.e. iOS 9.1 (13B143)
Created: 2015-10-27T19:42:42.432840
Originated: 2015-10-27T00:00:00
Open Radar Link: http://www.openradar.me/23281163
